pkg/apis/azure/storage/v1alpha1/types_test.go

```diff
@@ -47,14 +47,14 @@ var (
 func TestMain(m *testing.M) {
 	err := SchemeBuilder.AddToScheme(scheme.Scheme)
 	if err != nil {
-		log.Fatal(err)
+		logcore.Fatal(err)
```

Comment:
I have a recollection that I'd replaced this pattern with a combination of a `fmt.Println` and an `os.Exit`, but I can't find any evidence to support that. 🤔

Reply:
I too was wondering what we did for `log.Fatal` in our tests, to avoid the naming conflict with the `log` variable we are now creating.  We are still using `log.Fatal` in other tests, but those don't have a `log` variable also declared, so there's no conflict.  We'll need a nice long term solution that bails out of the tests like `log.Fatal` does, but doesn't create create a conflict between the imported `log` package and the `log` variable.  Open to suggestions, but considering just leaving this `logcore` alias for this PR.

Reply:
I've opened #389 to track this

Reply:
I think I weigh in on: 
```go
err := SchemeBuilder.AddToScheme(scheme.Scheme)
	if err != nil {
		log.Error(err, "failed to add to scheme")
		os.Exit(1)
	}
```